thirdeye-persistence/src/main/java/ai/startree/thirdeye/datalayer/bao/TaskManagerImpl.java

```diff
@@ -291,6 +293,41 @@ protected Long loadValue() {
         return count();
       }
     });
+
+    metricRegistry.register("taskLatencyInMillis", new CachedGauge<Long>(15, TimeUnit.SECONDS) {
```

Comment:
I think we should differentiate system latency for notification and detection tasks. 
It'll help when debugging.

Reply:
That's a great point! Will add separate metrics for both type of tasks.